src/components/Buttons/Button/types.ts

```diff
@@ -0,0 +1,18 @@
+export type ButtonSizeType = 'small' | 'medium' | 'large' | 'xLarge';
+
+export type ButtonTheme = 'solid' | 'secondary';
+
+export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
```

Comment:
1. ButtonHTMLAttributes로 지정해서 제네릭을 줘야 button태그의 모든 attr을 가져옵니다.
2. 
```suggestion
export interface ButtonProps extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, `aria-${string}`> {
```
요래해서 aria 관련 attr 제외할 수 있습니다! 자동완성시 굉장히 불편하더라구요...
추가로 접근성 관련한 기능은 컴포넌트 내에서 처리해 줘야 하는 부분이라고 생각됩니다!
같은 맥락으로 접근성 관련해서도 보완이 되면 좋을 것 같아요, 예를들면 탭 포커스시 아웃라인 표시 라던가 하는....
저희 remember-ui도 그렇고 대부분이 탭포커스시 별도 화면표시를 다 제거했는데 접근성 관점에서는 굉장히 좋지 않은 방법인디, rui부터는 가능한 부분에서 접근성 고려도 해보면 어떨까요?